tests/prague/eip7692_eof_v1/eip7620_eof_create/test_returncontract.py

```diff
@@ -0,0 +1,147 @@
+"""
+Tests for RETURNCONTRACT instruction validation
+"""
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.exceptions import EOFException
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-7620.md"
+REFERENCE_SPEC_VERSION = "f20b164b00ae5553f7536a6d7a83a0f254455e09"
+
+
+def test_returncontract_valid_index_0(
```

Comment:
mark that the test is valid from eof fork only like in other eof tests

```
from .. import EOF_FORK_NAME
...
pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
```

Reply:
@chfast only this comment and we can merge.